### PR TITLE
clip_

### LINKED ISSUE #22846: clip_
<-- 15039

### PR DESCRIPTION
**Pull request by NiteshK84**
# PR Description 
Adds clip_ func to paddle frontend

## Related Issue 

Close #22846

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by NiteshK84** [comment]
getting this CPU error, i think this is from ivy and not my changes, but please correct and guide me if this is not the case<
thanks!
image

File: ivy/functional/frontends/paddle/tensor/math.py
Comment (YushaArif99) [inline ivy/functional/frontends/paddle/tensor/math.py:1]: could you kindly revert the changes to other functions you've made in this file. You should only be adding the implementation for the `clip_` function

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @NiteshK84 

Extremely sorry for the delay in review 🙏🏼 . Dont know how this PR totally slipped under the radar as I normally try to be more vigilant about it and frequently view my notifications 😅 

With regards to the PR itself, could you kindly resolve the merge conflicts first so that the CI tests can run. Also kindly address the abovementioned point. Feel free to then re-request a review and I'll then proceed with a more detailed review. Thanks for your patience @NiteshK84 🙂
